examples/spring-boot-draft/README.md

```diff
@@ -0,0 +1,107 @@
+SpringBoot Demo for using Jib
+
+Tips: this is just a draft, please contribute if you have good suggestion.
+
+### Quickstart
+
+1.Modify configuration:
+
+Modify `CUSTOM_REGISTRY_URL`, `YOUR_USER_NAME`, `YOUR_PASSWORD`, `PASSWORD_ENCRYPT_BY_MAVEN` as yourself, and change `credHelper` to match your platform .
+
+2.Build the image:
```

Comment:
I'd recommend changing all the list numbering to `1.` - GitHub will generate the correct numbering.